src/server/seed.js

```diff
@@ -1 +1,59 @@
-// USMAN
\ No newline at end of file
+// USMAN
+import firebase from "firebase/compat/app";
+import "firebase/compat/firestore";
+import math from "mathjs";
+
+
+function Seed() {
+  const row = 3;
+  const col = 3;
+  console.log("WE got the seed function boys");
+
+  const dbh = firebase.firestore();
+
+
+  let objname = "";
+  let x = "";
+  let y = "";
+  let health = 1;
+  let id = "";
+
+  for (let r = 0; r < row; r++) {
+      const curCollection = dbh.collection(`row${r}`).doc(`master${r}`);
```

Comment:
Let's structure the database so that the row and the column are captured in the top level collection, i.e. (dbh.collection(ROW-COL).doc()). Also, we should have ~20 docs for each box (named 1, 2, 3, 4, etc) so we can shard writes across them. Each doc should then have an array that keeps track of the objects, so we can store several thousand objects per doc. The current implementation below stores just one object per doc which is very inefficient.